packages/flutter/test/material/radio_test.dart

```diff
@@ -356,6 +356,42 @@ void main() {
     );
   });
 
+  testWidgets('Radio with splash radius set', (WidgetTester tester) async {
+    tester.binding.focusManager.highlightStrategy = FocusHighlightStrategy.alwaysTraditional;
+    const double splashRadius = 30;
+    const Key radioKey = Key('radio');
+    Widget buildApp({bool enabled = true}) {
+      return MaterialApp(
+        home: Material(
+          child: Center(
+            child: StatefulBuilder(builder: (BuildContext context, StateSetter setState) {
+              return Container(
+                width: 100,
+                height: 100,
+                color: Colors.white,
+                child: Radio<int>(
+                  key: radioKey,
+                  value: 0,
+                  onChanged: (int? newValue) {},
+                  focusColor: Colors.orange[500],
+                  autofocus: true,
+                  groupValue: 0,
+                  splashRadius: splashRadius,
+                ),
+              );
+            }),
+          ),
+        ),
+      );
+    }
+    await tester.pumpWidget(buildApp());
+    await tester.pumpAndSettle();
+    expect(
+      Material.of(tester.element(find.byKey(radioKey))),
```

Comment:
We could dispense with the key and just use find.byType(Radio)? Maybe the type parameter makes that problematic?

Reply:
As you say its the type parameter makes it problematic: https://github.com/dart-lang/sdk/issues/11923

I can change it to `find.byWidgetPredicate((Widget widget) => widget is Radio<int>)` to remove the key.